src/content/en/updates/2019/03/prefers-reduced-motion.md

```diff
@@ -0,0 +1,231 @@
+While many users enjoy such animations, some users dislike
+them because they feel distracted or slowed down from them. In the worst case, users may even suffer
+from motion sick as if it were a real life experience, so for these users reducing animations is a
+*medical necessity*.
+
+## Motion-triggered vestibular spectrum disorder
+
+Some users experience distraction or nausea from animated content. For example, if scrolling a page
+causes elements to move other than the essential movement associated with scrolling—as with parallax
+scrolling, where backgrounds move at a different rate to foregrounds—it can trigger vestibular
+disorders. Vestibular (inner ear) disorder reactions include dizziness, nausea and headaches.  The
+[impact of animation on people with vestibular disorders](https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html)
+can be quite severe. Triggered reactions include nausea, migraine headaches, and potentially needing
+bed rest to recover.
+
+## Remove motion on operating systems
+
+Operating systems like Android, iOS, macOS, or Windows in their accessibility settings have allowed
+users for a long time to reduce motion wherever possible. The screenshots below show Android Pie's
+"remove animations" preference and macOS Mojave's "reduce motion" preference that, when checked,
+cause the particular operating systems to not use decorative effects like app launching animations.
+Applications themselves can and should honor this setting, too, and remove all unnecessary
+animations.
+
+<img src="../../images/2019/03/prefers-reduced-motion/android-remove-animations.png" intrinsicsize="472x493" width="287" height="300" alt="Android settings screen with 'remove animations' checkbox checked">
+<img src="../../images/2019/03/prefers-reduced-motion/macos-reduce-motion.png" intrinsicsize="780×588" width="398" height="300" alt="macOS settings screen with 'reduce motion' checkbox checked">
+
+## Remove motion on the web
+
+[Media Queries Level&nbsp;5](https://drafts.csswg.org/mediaqueries-5/)
+brings this user preference to the web as well. Media queries allow authors to test and query values
+or features of the user agent or display device, independent of the document being rendered.
+The media query
+[`prefers-reduced-motion`](https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion)
+is used to detect if the user has requested the system minimize the amount of animation or motion it
+uses. It can take two possible values:
+
+- `no-preference`: Indicates that the user has made no preference known to the system.
+  This keyword value evaluates as `false` in the boolean context.
+- `reduce`: Indicates that the user has notified the system that they prefer an interface that
+  minimizes the amount of movement or animation, preferably to the point where all non-essential
+  movement is removed.
+
+## Working with the media query
+
+Note: `prefers-reduced-motion` is available as of Chrome Canary&nbsp;74.
```

Comment:
Is it worth noting anywhere (maybe here, maybe not) that this is also available on Firefox and Safari (e.g. it's cross-browser compatible), or is that out of scope for this article?

Reply:
Added link to https://caniuse.com/#feat=prefers-reduced-motion.